src/ashley/machina_extensions/forum_conversation/migrations/0014_auto_20210302_1614.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 3.1.7 on 2021-03-02 16:14
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("forum_conversation", "0013_auto_20201220_1745"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="topic",
+            name="posts_count",
+            field=models.PositiveIntegerField(
+                blank=True,
+                db_index=True,
+                default=0,
+                editable=False,
+                verbose_name="Posts count",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="topic",
+            name="views_count",
+            field=models.PositiveIntegerField(
+                blank=True,
+                db_index=True,
+                default=0,
+                editable=False,
+                verbose_name="Views count",
+            ),
+        ),
+    ]
```

Comment:
I understand that you are adding this migration to add an index on these fields.
I'm a bit scared this will complicate our life if django-machina adds migrations. Maybe this militates for trying to sell it to them first and trying to integrate it in django-machina? If it doesn't work we can always come back to this PR?

Reply:
In the real PR for machina I won't have to overload anything, I can directly add in the model the index so it won't look so massive. I will have only a file for the makemigration or if you want to I can even not add this modification on the PR for machina but for our project with the amount of forums and expected posts that we have and classes, it's a point that needs to be optimized IMO and as it's already done, I'm not certain to understand why it shouldn't be used

Reply:
For other django-machina apps that we have overloaded, we copied the migration files.
We just have to run the `makemigrations` command to stay up-to date with django-machina migrations, right ? (except if they write migrations manually).

Reply:
@sampaccoud To better understand, you suggest to submit this whole multi-sort feature to django-machina ? Or just the index part ?

Reply:
Discussing this with Sam, we've decided to go ahead in our own repo and first validate this PR then do a PR on Django Machina which we hope will be accepted.
Then when we'll update our repo with the new version of Django machina containing the sorting, to get rid of the unnecessary code on our side. 
As Sam is quite sure it will get approved, his idea was not to override the model and add index in our repo because that could get messy once our PR gets accepted on Django machina

Reply:
@madmatah don't you think it's an interesting feature for the original project?